src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -86,6 +93,7 @@ export class IndexPattern implements IIndexPattern {
   // savedObject version
   public version: string | undefined;
   public sourceFilters?: SourceFilter[];
+  public dataSourceRef?: SavedObjectReference;
```

Comment:
seems some other changes is added to this PR, can you clean it up?

Reply:
Upstream had a force push that leads to some diffs. I'll fix it